lib/phoenix_live_view/js.ex

```diff
@@ -101,6 +101,38 @@ defmodule Phoenix.LiveView.JS do
   >+</button>
   ```
 
+  ## DOM Selectors
+
+  The client utility commands in this module all take an optional DOM selector
+  using the `:to` option.
+
+  This can be a string for a regular DOM selector such as:
+
+  ```elixir
+  JS.add_class("warmer", to: ".thermo")
+  JS.hide(to: "#modal")
+  JS.show(to: "body a:nth-child(2)")
+  ```
+
+  It is also possible to provide scopes to the DOM selector. The following scopes
+  are available:
+
+   * `{:inner, "selector"}` To target an element within the interacted element.
+   * `{:closest, "selector"}` To target the closest element within the interacted
```

Comment:
```suggestion
   * `{:closest, "selector"}` To target the closest element from the interacted element upwards.
```

this is basically https://developer.mozilla.org/en-US/docs/Web/API/Element/closest